runtime/onert/core/src/exec/train/TrainableExecutor.h

```diff
@@ -90,6 +91,7 @@ class TrainableExecutor : public IExecutor
 
 private:
   std::vector<compiler::train::TrainableCodeAndInfo> _code;
+  std::vector<compiler::train::TrainableCodeAndInfo> _bcode;
```

Comment:
I don't understand why `_bcode` is needed. Currently `TrainableFnSequence` already has `backward()`. What is the difference?

Reply:
The `TrainableExecutor` uses `_code` to call the `forward()` and `backward()` functions. The `_code` contains the order in which operation is called. This commit introduces `_bcode` for backwarding order. The current onert calls the backwarding function in the reverse direction of `_code` order. However, the backwarding should start with `Loss` operator. So this pr searches for a new order for backwarding.

Reply:
I don't understand fully in detail. Could you please show me the full draft, which uses the added members and methods?

Reply:
There is no draft code. This PR works completely with this PR alone. Use of this member can be found in the `TrainableExecutor.cc` file.

Reply:
I think you want to traverse in backward order for the same code in `_code`. Then, backward order iterator is what you need as I understand. Is it right?

If so, we may add `iterate()` and `biterate()` in `TrainableCodeMap` (preferred but it is currently just an alias) or `TrainableCodeAndInfo`.

Reply:
I wrote here I explained offline:

https://github.com/Samsung/ONE/blob/e48b702468fc683deafbbde69edb960ad7eb2638/runtime/onert/core/include/util/ObjectManager.h#L186-L192

You can introduce `back_iterate`. In this method, you can make backward ordering and iterate and call your function.